python-threatexchange/threatexchange/signal_type/pdq.py

```diff
@@ -69,14 +70,11 @@ def compare_hash(
         dist = simple_distance(hash1, hash2)
         return signal_base.HashComparisonResult.from_dist(dist, thresh)
 
-    @classmethod
-    def hash_from_file(cls, file: pathlib.Path) -> str:
-        pdq_hash, _quality = pdq_from_file(file)
-        return pdq_hash
-
     @classmethod
     def hash_from_bytes(cls, bytes_: bytes) -> str:
-        pdq_hash, _quality = pdq_from_bytes(bytes_)
+        pdq_hash, quality = pdq_from_bytes(bytes_)
+        if quality < cls.QUALITY_THRESHOLD:
```

Comment:
Unsure: it might be worth a warning (or at least debug log) here as this would be a hard thing to debug. Based on `hash_from_bytes` it is within the supported behavior but still feels like a "fail quietly" choice.

Reply:
Same problem with tlsh - it needs at least 100 bytes or it returns empty. Punting into the future.